salt/files/cloud.profiles.d/proxmox.conf

```diff
@@ -0,0 +1,19 @@
+# This file managed by Salt, do not edit by hand!!
+{% set cloud = salt['pillar.get']('salt:cloud', {}) -%}
+{% if 'proxmox_profile_defaults' in cloud %}
+{% set defaults = cloud.get('proxmox_profile_defaults') %}
+
+prox-test-1:
+    provider: proxmox-test
+    image: ISO:vztmpl/debian-7.0-standard_7.0-2_i386.tar.gz
+    technology: openvz
+    host: testvm01
+    cpus: 2
+    memory: 2048
+    swap: 2048
+    disk: 5
+    password: {{ defaults.get('default_password', 'password') }}
```

Comment:
I don't know if "default password" is quite the right name. I just didn't want to put a password in my file server tree. Perhaps there's a better name? Or just leave the password in the template and assume you use a non-sensitive one at this point and change it with the highstate later.